bcipy/signal_model/mach_learning/m_estimator/m_estimator.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import scipy as sc
+
+
+def eigsorted(cov):
+    """
+    Find and sort eigen values and vectors
+    :param cov: covariance matrix
+    :return: sorted eigen values and vectors
+    """
+
+    vals, vecs = np.linalg.eigh(cov)
+    order = vals.argsort()[::-1]
+    return vals[order], vecs[:, order]
+
+
+def u_func(t, b, c_square):
+    """
+    Huber's loss function
+    :param t: Mahalanobis distance
+    :param b: a constant in p
```

Comment:
what are p and q? define

Reply:
![image](https://user-images.githubusercontent.com/30954743/41722020-5c2afa58-7535-11e8-95e2-aab1c70c0cbf.png)

![image](https://user-images.githubusercontent.com/30954743/41722038-6850b43a-7535-11e8-9d95-b8954cd4667e.png)